<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>pyFluo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Arduino port</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="comLineEdit">
        <property name="text">
         <string>COM6</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Second LED</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="LEDcomboBox2">
          <property name="currentText">
           <string>365 nm</string>
          </property>
          <item>
           <property name="text">
            <string>365 nm</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>385 nm</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>470 nm</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>First LED</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="LEDcomboBox1">
          <item>
           <property name="text">
            <string>365 nm</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>385 nm</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>470 nm</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Third LED</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="LEDcomboBox3">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="currentIndex">
           <number>2</number>
          </property>
          <item>
           <property name="text">
            <string>365 nm</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>385 nm</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>470 nm</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Select LEDs</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="modecomboBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>1 wavelength</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2 wavelengths</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3 wavelengths</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Exposure 1st wavelength (ms)</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QSpinBox" name="exp1Spin">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100000</number>
        </property>
        <property name="value">
         <number>20</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Exposure 2nd wavelength (ms)</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QSpinBox" name="exp2Spin">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100000</number>
        </property>
        <property name="value">
         <number>15</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Trigger event between</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="eventStartSpinBox"/>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>and</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="eventEndSpinBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Interval between exposures (ms)</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QSpinBox" name="isiSpin">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100000</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>InterFrame interval (ms, &gt;Exp1+Exp2+Interval)</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QSpinBox" name="ifiSpin">
        <property name="maximum">
         <number>100000</number>
        </property>
        <property name="value">
         <number>1000</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="Led1PushButton">
        <property name="text">
         <string>LED 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Led2PushButton">
        <property name="text">
         <string>LED 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="eventpushButton">
        <property name="text">
         <string>Event</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="blinkPushButton">
        <property name="text">
         <string>Start Protocol</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
